pom.xml

```diff
@@ -231,6 +241,14 @@
 				</plugin>
 			</plugins>
 		</pluginManagement>
+		<resources>
```

Comment:
Are you sure this exclusion needs to go back in?

This was making all static files be excluded from war package for me.